app/views/ExploreDeep/ActionCell/index.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useCallback } from 'react';
+import { _cs } from '@togglecorp/fujs';
+import {
+    Button,
+    ConfirmButton,
+    useAlert,
+} from '@the-deep/deep-ui';
+import { useMutation, gql } from '@apollo/client';
+
+import { useModalState } from '#hooks/stateManagement';
+import {
+    CancelJoinProjectMutation,
+    CancelJoinProjectMutationVariables,
+} from '#generated/types';
+
+import ProjectJoinModal from './ProjectJoinModal';
+
+import styles from './styles.css';
+
+export interface Props {
+    className?: string;
+    projectId: string;
+    disabled?: boolean;
+    membershipPending: boolean;
```

Comment:
Let's add a note to indicate that this is not actual data pending for membership